src/page/template/list/start-ui.htm

```diff
@@ -124,10 +124,24 @@
                 <!-- /ko -->
                 <group-list params="groups: searchResults.groups, click: clickOnConversation"></group-list>
               </div>
-              <div class="others" data-bind="visible: searchResults.others().length > 0">
-                <span class="start-ui-list-header" data-bind="text: enableFederation ? t('searchOthersFederation', federationDomain) : t('searchOthers')"></span>
-                <user-list class="search-list-theme-black" params="user: searchResults.others, click: clickOnOther, mode: UserlistMode.OTHERS, searchRepository: searchRepository, teamRepository: teamRepository, conversationRepository: conversationRepository"></user-list>
-              </div>
+              <!-- ko if: searchOnSameFederatedDomain() -->
+                <div class="others" data-bind="visible: searchResults.others().length > 0">
+                  <span class="start-ui-list-header" data-bind="text: t('searchOthersFederation', federationDomain)"></span>
+                  <user-list class="search-list-theme-black" params="user: searchResults.others, click: clickOnOther, mode: UserlistMode.OTHERS, searchRepository: searchRepository, teamRepository: teamRepository, conversationRepository: conversationRepository"></user-list>
+                </div>
+              <!-- /ko -->
+              <!-- ko if: searchOnOtherFederatedDomain() -->
+                <div class="others" data-bind="visible: searchResults.others().length > 0">
+                  <span class="start-ui-list-header" data-bind="text: t('searchOtherDomainFederation')"></span>
```

Comment:
To me it looks like it's 3x the same template but just with another text in the `start-ui-list-header`. Can we just wrap the headers in `ko if` statements?